ghost/core/core/server/data/migrations/versions/5.19/2022-10-10-06-58-add-subscriptions-table.js

```diff
@@ -0,0 +1,19 @@
+const {addTable} = require('../../utils');
+
+module.exports = addTable('subscriptions', {
+    id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+    type: {type: 'string', maxlength: 50, nullable: false},
+    status: {type: 'string', maxlength: 50, nullable: false},
+    member_id: {type: 'string', maxlength: 24, nullable: false, unique: false, references: 'members.id', cascadeDelete: true},
+    tier_id: {type: 'string', maxlength: 24, nullable: true, unique: false, references: 'products.id'},
+    cadence: {type: 'string', maxlength: 50, nullable: true},
+    currency: {type: 'string', maxLength: 3, nullable: true},
```

Comment:
This is a bug - should be `maxlength` but it turns out we also make the same mistake elsewhere. I'm also not sure if we should be using `3` because the places where we get it right, we use `50`. The only problem is those using `maxLength` are set to 191

```
./schema.js
447:        currency: {type: 'string', maxlength: 50, nullable: true},
465:        currency: {type: 'string', maxlength: 50, nullable: true},
531:        currency: {type: 'string', maxLength: 3, nullable: false},
580:        currency: {type: 'string', maxLength: 3, nullable: false},
639:        plan_currency: {type: 'string', maxLength: 3, nullable: false}
689:        currency: {type: 'string', maxLength: 3, nullable: false},
```

@allouis What would you prefer to do here in terms of length for this new column? I can write up a tech debt issue for fixing these other ones

Reply:
Eeesh! That's a tricky one!

In terms of the new column, I think 50 is what we should go for, 3 was probably a mistake in hindsight

Reply:
Ok cool 👍🏻 

My plan to fix `maxLength` is to switch it to `maxlength: 191`, which is effectively what it resolved to, and then we can reduce it down if we want to in the future